packages/lorcana-cards/src/cards/010/characters/045-elsa-exploring-the-unknown.ts

```diff
@@ -0,0 +1,45 @@
+import type { CharacterCard } from "@tcg/lorcana";
+
+export const elsaExploringTheUnknown: CharacterCard = {
+  id: "744",
+  cardType: "character",
+  name: "Elsa",
+  version: "Exploring the Unknown",
+  fullName: "Elsa - Exploring the Unknown",
+  inkType: ["amethyst"],
+  franchise: "Frozen",
+  set: "010",
+  text: "CLOSER LOOK When you play this character, you may draw a card.",
+  cost: 3,
+  strength: 1,
+  willpower: 3,
+  lore: 1,
+  cardNumber: 45,
+  inkable: true,
+  externalIds: {
+    ravensburger: "19a46ebcde9538732365e630c3cd5a56fd6ec603",
+  },
+  abilities: [
+    {
+      id: "744-1",
+      text: "CLOSER LOOK When you play this character, you may draw a card.",
+      name: "CLOSER LOOK",
+      type: "triggered",
+      trigger: {
+        event: "play",
+        timing: "when",
+        on: "SELF",
+      },
+      effect: {
+        type: "optional",
```

Comment:
Similar to https://github.com/TheCardGoat/tcg-engines/pull/101/changes#r2605225879

I have the feeling that this could be a boolean property of an effect / ability. What do you think?